src/routes.json

```diff
@@ -181,6 +181,13 @@
         "title": "Jobs"
     },
     {
+        "name": "jobs-moderator",
+        "pattern": "^jobs/moderator/?$",
+        "routeAlias": "/jobs/moderator/?$",
```

Comment:
I think we want the `routeAlias` to simply be `/jobs` (see "microworlds" pattern below).